main.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"fmt"
+	"github.com/luciodesimone/golang-bootcamp/beer"
+	"github.com/luciodesimone/golang-bootcamp/db"
+	"os"
+)
+
+func main() {
+	stout := beer.New("Creamy black dense beer with coffee notes, ideal before dinner", 42, 8, 40, 92)
+	bitter := beer.New("dark amber light beer, ideal for summer", 57, 5.5, 10, 63)
+	barley := beer.New("Strong, dense red/dark beer, ideal for late night with friends", 70, 12, 22, 86)
+	newB := beer.Beer{}
+
+	//you must provide a valid json file
+	conn, err := db.Open("beers.json")
+
+	if err != nil {
+		fmt.Printf("Error creating file: %s\n", err)
+		os.Exit(1)
+	}
+
+	conn.CreateBeer(&stout)
+	conn.CreateBeer(&bitter)
+	conn.CreateBeer(&barley)
+
+	err = conn.GetBeer(stout.ID, &newB)
+
+	if err != nil {
+		fmt.Printf("Error getting beer: %s\n", err)
+	} else {
+		fmt.Printf("Beer!: %v\n", newB)
+	}
+
+	defer conn.Close()
```

Comment:
This could go on line 23 to be more in-line with effective go best practices.

The defer is a good choice and makes sense :nerd_face:  (read my comment above about defering the file close on the main), the only thing here is, defer as soon as you know the resource was opened successfully.